_posts/2015-10-27-drupal-web-mapping.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Drupalized Web Mapping @ NACIS 2015"
+date: 2015-10-27
+author: Tim Stallmann
+tags: Drupal, Drupal Planet, Cartography
```

Comment:
these should be: `drupal drupal-planet cartography`, see `tags.yml` in this repo